### PR TITLE
Create contact form

### PR DESCRIPTION
**Pull request by aimalamiri**
Created the contact form, and also the footer section is designed with its icons and bottom line.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,234 +2,320 @@
...
+          like to chat please get in touch.
+        </h4>
+      </div>
+      <form action="https://formspree.io/f/mzborevw" method="post" class="contact-form">
+        <input type="text" name="name" class="input-field font-inter" placeholder="Full name" maxlength="30" required />
+        <input type="email" name="email" class="input-field font-inter" placeholder="Email address" required />
+        <textarea name="message" class="input-field input-textarea font-inter" maxlength="500" required>
+Enter text here</textarea>
+        <button class="button contact-form-button mt-24">Get in touch</button>
+      </form>
```
Comment (Felipeg005) [inline index.html:283]: You did a great implementation here with your form, Great Job! 👍

File: styles.css [hunk 2]
```diff
@@ -263,6 +263,65 @@ header {
...
+  margin-top: 15px;
+  margin-bottom: 13px;
+}
+
+.footer-line-section {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 8px;
+}
+
```
Comment (Felipeg005) [inline styles.css:324]: Your form in the mobile version it's looking great. 💯

**Review by Felipeg005 — APPROVED** [review]
Hi @aimalamiri , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (aimalamiri) [reply styles.css:324]: Thanks, @Felipeg005.

**Pull request merged**